### ISSUE TITLE
Tuning an underwater acoustic link

### ISSUE THREAD

**Issue by prasadtiru** [org-arl/unet-blog#7]
The intent of this post is to touch upon the link tuning techniques and how such techniques can be implemented in underwater acoustic modems.

**Comment by mchitre** [comment]
Include FDMA or deconflicting frequency bands in the same post?

**Comment by manuignatius** [comment]
Yes, please also include how to avoid overlapping frequency bands from other devices in this post.

**Comment by manuignatius** [comment]
Frequency band control is covered in this blog: https://blog.unetstack.net/frequency-band-control-in-unetstack
